python/scripts/generate_openapi.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -ex
+
+cd svix
+
+if [ ! -d "./internal" ]; then
+    openapi-python-client generate --path ../../openapi.json --custom-template-path=../templates --config ../openapi-generator-config.json
+else
+    openapi-python-client update --path ../../openapi.json --custom-template-path=../templates --config ../openapi-generator-config.json
+fi
```

Comment:
This is fine, but it's a bit of duplication (if you update one path you need to remember to do both), rather than if-ing on the whole function you can just conditionally set a variable to the command then run
`openapi-python-client $COMMAND --path ...`